src/components/PostRelatedArticles/index.jsx

```diff
@@ -0,0 +1,71 @@
+import { graphql, useStaticQuery } from "gatsby"
+import React from "react"
+
+import * as styles from "./index.module.scss"
+
+export const query = graphql`
+  {
+    allMarkdownRemark(
+      filter: { fileAbsolutePath: { regex: "/src.posts/" } }
+      sort: [{ frontmatter: { date: DESC } }, { frontmatter: { title: DESC } }]
+    ) {
+      nodes {
+        fields {
+          path
+        }
+        frontmatter {
+          categories {
+            key
+            label
+          }
+          title
+        }
+      }
+    }
+  }
+`
+
+function Article({ article }) {
+  return (
+    <div className={styles.article} key={article.fields.path}>
```

Comment:
I think we should make the whole card clickable

Reply:
And we should probably use `Link` for navigating internal pages: https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-link/